### DISCUSSION TITLE
CORS error using apollo client from react

### DISCUSSION THREAD

**Discussion by vikramdewangan** [PokeAPI/pokeapi#776] [Q&A]
The service is blocked by CORS when accessed through @apollo/client with graphql packages from react localhost:3000. 
The same setup works with [https://graphql.anilist.co/](https://graphql.anilist.co/)
The problem is if **mode: "no-cors"** is passed the fetch payload won't be sent as application/json and rather would be sent as text/plain which gives 502.

Please add a CORS policy for Access-Control-Allow-Origin: *

CC: @phalt

**Comment by Naramsim** [comment]
Hi! Can you please retry? GraphQL was down

**Reply by vikramdewangan** [reply]
Seems working fine now, thanks a lot @Naramsim .
